sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -426,7 +426,7 @@ def init_worker(self, task):
         engine_logpath = (
             os.path.dirname(logpath)
             + "/engine/"
-            + os.getenv("SLURM_JOB_NAME")
+            + os.getenv("SLURM_JOB_NAME", self.process_task_name(task.task_name()))
```

Comment:
this might get the correct path again when the env var is not set, but will the worker write the log into that file if it is not started via slurm? Would it not be printed into stdout/stderr of the terminal?

Reply:
> Would it not be printed into stdout/stderr of the terminal?

Yes it prints on stdout.

But this code crashes with an exception, because it tries to do `"...some string..." + None`.